lntest/harness.go

```diff
@@ -346,10 +334,15 @@ func (n *NetworkHarness) NewNodeEtcd(name string, etcdCfg *etcd.Config,
 // NewNode fully initializes a returns a new HarnessNode bound to the
 // current instance of the network harness. The created node is running, but
 // not yet connected to other nodes within the network.
-func (n *NetworkHarness) NewNode(name string, extraArgs []string) (*HarnessNode,
-	error) {
+func (n *NetworkHarness) NewNode(t *testing.T,
```

Comment:
nit: we could pass `t *harnessTest` to `NewNode` and then we wouldn't have to write `NewNode(t.t ...)` in tests.

Reply:
See comments below.